app/src/ui/chairlift_data.py

```diff
@@ -1,18 +1,17 @@
 from dataclasses import dataclass
 from typing import List
+from snowflake.snowpark import Session
 
 import streamlit as st
-from snowflake.snowpark.context import get_active_session
-session = get_active_session()
 
 @dataclass
 class Machine:
     uuid: str
     name: str
 
 @st.cache_data
-def get_machines() -> List[Machine]:
-    machine_tuples = session.sql(f"""
+def get_machines(_session: Session) -> List[Machine]:
```

Comment:
nit: why the underscore now?

Reply:
looks like streamlit can't cache session (hash it) so it asks me to add underscore to not cache it.

Reply:
ah alright, that makes sense